Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN dnf install -y python3 python3-requests && \
 RUN mkdir -p /var/lib/ironic-inspector && \
     sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal"
 
+RUN dnf remove -y sqlite
```

Comment:
nit: maybe merge into the previous `RUN` to reduce the number of layers?

Reply:
yeah.... I completely ignored that for some reason